tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/decoder/DoubleRLBEDecoder.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iotdb.tsfile.encoding.decoder;
+
+import org.apache.iotdb.tsfile.file.metadata.enums.TSEncoding;
+
+import java.nio.ByteBuffer;
+
+public class DoubleRLBEDecoder extends Decoder {
+  /** constructor of DoubleRLBEDecoder */
+  public DoubleRLBEDecoder() {
+    super(TSEncoding.RLBE);
+    numberLeftInBuffer = 0;
+    byteBuffer = 0;
+  }
+
+  /** how many numbers expected to decode in following block */
+  private int blocksize;
+
+  /** origin values are stored in data */
+  private double[] data;
+
+  /** whether the first value is decoded */
+  private int writeindex = -1;
+
+  /** the pointer of current last value */
+  private int readindex = -1;
+
+  /** fibonacci values are stored in fibonacci */
+  private long[] fibonacci;
```

Comment:
Fibonacci numbers are mathematically constant so it is not necessary for each decoder to have a separate copy. Maybe you can make it a static field and initialize it properly at the system startup.